controllers/admin/AdminGroupsController.php

```diff
@@ -163,7 +163,7 @@ public function initPageHeaderToolbar()
     {
         if (Group::isFeatureActive() && empty($this->display)) {
             $this->page_header_toolbar_btn['new_group'] = [
-                'href' => self::$currentIndex . '&addgroup&token=' . $this->token,
+                'href' => $this->context->link->getAdminLink('AdminGroups') . '&addgroup&token=' . $this->token,
```

Comment:
```suggestion
                'href' => $this->context->link->getAdminLink('AdminGroups', true, [], ['addgroup' => 1]),
```

It's better to use parameters, cleaner and allows us to properly migrate that link to a symfony one, automatically, if that page is migrated.

Reply:
Fixed @Hlavtox :)